src/js/components/Calendar.js

```diff
@@ -285,24 +300,44 @@ export default class Calendar extends Component {
   }
 
   render () {
-    var classes = [CLASS_ROOT];
-    if (this.state.dropActive) {
-      classes.push(CLASS_ROOT + "--active");
-    }
-    if (this.props.className) {
-      classes.push(this.props.className);
-    }
+    const {
+      className,
+      id,
+      name,
+      value,
+      onChange, // eslint-disable-line no-unused-vars
+      ...props
+    } = this.props;
+
+    const classes = classnames(
+      CLASS_ROOT,
+      className,
+      {
+        [`${CLASS_ROOT}--active`]: this.state.dropActive
+      }
+    );
 
     return (
-      <div ref="component" className={classes.join(' ')}>
-        <input className={CLASS_ROOT + "__input"}
-          id={this.props.id} ref="calendarInput" name={this.props.name}
-          value={this.props.value}
-          onChange={this._onInputChange} />
-        <Button className={CLASS_ROOT + "__control"} icon={<CalendarIcon
-          a11yTitle='calendar-icon-title'
-          a11yTitleId='calendar-icon-title-id' />}
-          onClick={this._onOpen} />
+      <div ref="component" className={classes}>
+        <input
+          className={`${CLASS_ROOT}__input`}
+          id={id}
+          ref="calendarInput"
+          name={name}
+          value={value}
+          onChange={this._onInputChange}
+          {...props}
```

Comment:
Should we pass the additional props first, before the className, just in case there's a duplicate for another prop at some point? I don't think there's an actual issue with this current set of props but I've found that to be a more defensive way of programming in case additional props are added later.